easybuild/easyconfigs/g/GLIMPSE/GLIMPSE-2.0.0-GCC-13.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'MakeCp'
+
+name = 'GLIMPSE'
+version = '2.0.0'
+
+homepage = 'https://github.com/odelaneau/GLIMPSE'
+description = "GLIMPSE2 is a set of tools for phasing and imputation for low-coverage sequencing datasets"
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/odelaneau/GLIMPSE/archive/']
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/odelaneau/',
+        'repo_name': 'GLIMPSE',
+        'tag': 'v%(version)s',
+        'keep_git_dir': True,
```

Comment:
do we need this? seems to build fine for me without the git dir, and then we could actually checksum it